Prox/Prox/Models/Place.swift

```diff
@@ -451,6 +451,37 @@ struct OpenHours {
         return nil
     }
 
+    func getEventTimeText(forTime baseTime: Date) -> (String, String)? {
+        // Note: this function is basically a modified version of nextOpeningTime()
+        let midnight = DateComponents(hour: 0, minute: 0)
+
+        // HACK: Keeping this as a range so we can easily change this function look ahead more days if needed.
+        // (1...N) will look ahead N days after today.
+        let times = [baseTime] + (1...1).map { getDate(forTime: midnight, onDate: baseTime.addingTimeInterval(TimeInterval($0) * AppConstants.ONE_DAY))! }
```

Comment:
nit: -> `todayAndTomorrow`, or something. It's unclear which times these are and how we intend to use them.